sdk/storage/azure-storage-queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -482,10 +488,10 @@ public URL getQueueUrl() {
      * @return The first {@link DequeuedMessage} in the queue, it contains
      * {@link DequeuedMessage#messageId() messageId} and {@link DequeuedMessage#popReceipt() popReceipt} used to interact
      * with the message, additionally it contains other metadata about the message.
-     * @throws StorageErrorException If the queue doesn't exist
+     * @throws StorageException If the queue doesn't exist
      */
     public Flux<DequeuedMessage> dequeueMessages() {
-        return dequeueMessages(1, Duration.ofSeconds(30));
+        return dequeueMessages(1, null);
```

Comment:
isn't this a backwards compatibility break against preview 2?  do we care about it?  Behavior is changing to be different after this right?

Reply:
This is the correct behavior, that code was adding a REST API specification into our client. We should avoid those whenever possible.

https://docs.microsoft.com/en-us/rest/api/storageservices/get-messages#uri-parameters